db/people.yml

```diff
@@ -561,15 +561,12 @@ stao:
       department: apam
       group: bg
       location: New York, NY
-      other:
         - 'Adviser: Simon Billinge'
     - begin_year: 2013
       degree: BS in Physics
       end_year: 2017
       institution: Peking University
       location: Beijing, China
-      other:
```

Comment:
it is correct to delete this one, because you don't have anything to add under "other", like a minor, so you don't need the ``other`` field.